src/backend/db/users.js

```diff
@@ -9,10 +9,10 @@ import { formatDate } from "../utils/authUtils";
 export const users = [
   {
     _id: uuid(),
-    firstName: "Adarsh",
-    lastName: "Balika",
-    email: "adarshbalika@gmail.com",
-    password: "adarshbalika",
+    firstName: "Rohan",
+    lastName: "Dubey",
+    email: "rohandubey154@gmail.com",
+    password: "rohan123",
```

Comment:
Prefer sticking to dummy data, hopefully this email/password is not real and used